src/site/content/en/blog/coop-coep/index.md

```diff
@@ -274,6 +278,38 @@ To use COOP Reporting API, the token must be served as an HTTP header instead of
 a `<meta>` tag.
 {% endAside %}
 
+#### Set up a server to receive reports {: #set-up-reporting-endpoint}
+
+A server with reporting endpoints needs to be set up in order to receive reports
+from your COOP/COEP. There are two options:
+
+* Use a solution that accepts reports.
+* Build your own server that accepts reports.
+
+##### Use a solution that accepts reports
+
+We are aware of one solution that accepts reports from the browser's COOP/COEP reporting functionality.
+
+* [https://uriports.com](https://uriports.com)
+
+If there's any other solutions that accept reports, [please let us know to
+update this post](https://github.com/GoogleChrome/web.dev).
```

Comment:
@agektmr @mihajlija we can leave this but I want to give you a heads up that it may become a problem and I might need to remove the section

The problem is that if we highlight one paid service but neglect to highlight other paid services, it makes it look like we're picking favorites and it's a hard position to justify

In the interest of providing developers a clear path to getting reporting set up I am OK with leaving it in for now

Reply:
This was something I was on the fence. I know one more similar service (https://report-uri.com) but they don't support COOP/COEP yet. There are no other equivalents as far as I know.